### PR TITLE
All: refactor - rename CanalTP to hove-io

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: release/script_release.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def get_tag_name(version):
 
 
 class ReleaseManager:
-    def __init__(self, release_type, remote_name="canalTP"):
+    def __init__(self, release_type, remote_name="hove-io"):
```
Comment (woshilapin) [inline release/script_release.py]: Not sure about that one. If someone can confirm it's ok?

Comment (xlqian) [reply release/script_release.py]: it's the name of git remote on local machine. if it fails on run time, no worries

**Review by xlqian — APPROVED** [review]

**Comment by xlqian** [comment]
you forgot to change the string in tests LOL

**Comment by woshilapin** [comment]
> you forgot to change the string in tests LOL

Yes, I did... on purpose. If it ain't broken, don't fix it :stuck_out_tongue: That said, I was less confident to change tests :smile: and it's not a problem to keep Kisio in the fixtures and tests I believe.

**Comment by woshilapin** [comment]
I'm gonna extract some part of this PR into smaller (or dumber PR). First one already opened is #3712.

File: source/jormungandr/setup.py [hunk 2]
```diff
@@ -35,7 +35,7 @@
 setup(
     name='jormungandr',
     description='webservice d\'exposition en http de kraken',
-    author='Kisio Digital',
+    author='Hove',
     author_email='navitia.pt.core@kisio.com',
```
Comment (pbougue) [inline source/jormungandr/setup.py:39]:
```suggestion
    author_email='core@hove.com',
```

File: source/monitor/setup.py [hunk 3]
```diff
@@ -36,7 +36,7 @@
     name='monitor_kraken',
     version='0.39.0',
     description="Provide a small api for monitoring kraken",
-    author='CanalTP',
+    author='Hove',
     author_email='alexandre.jacquin@canaltp.fr',
```
Comment (pbougue) [inline source/monitor/setup.py:40]:
```suggestion
    author_email='core@hove.com',
```

File: source/navitiacommon/setup.py [hunk 4]
```diff
@@ -34,7 +34,7 @@
 setup(
     name='navitiacommon',
     description='module shared between jormungandr and tyr',
-    author='CanalTP',
+    author='Hove',
     author_email='alexandre.jacquin@canaltp.fr',
```
Comment (pbougue) [inline source/navitiacommon/setup.py:38]:
```suggestion
    author_email='core@hove.com',
```

File: source/tyr/setup.py [hunk 5]
```diff
@@ -35,7 +35,7 @@
 setup(
     name='tyr',
     description="Provide an API for managing users and handle data updates",
-    author='CanalTP',
+    author='Hove',
     author_email='alexandre.jacquin@canaltp.fr',
```
Comment (pbougue) [inline source/tyr/setup.py:39]:
```suggestion
    author_email='core@hove.com',
```

**Review by pbougue — APPROVED** [review]
With or without suggestions, it's an improvement :)

File: source/navitiacommon/setup.py (hunk 4 above)
Comment (woshilapin) [reply source/navitiacommon/setup.py:38]: See https://github.com/hove-io/navitia/pull/4017.

File: source/monitor/setup.py (hunk 3 above)
Comment (woshilapin) [reply source/monitor/setup.py:40]: See https://github.com/hove-io/navitia/pull/4017

File: source/jormungandr/setup.py (hunk 2 above)
Comment (woshilapin) [reply source/jormungandr/setup.py:39]: See https://github.com/hove-io/navitia/pull/4017

File: source/tyr/setup.py (hunk 5 above)
Comment (woshilapin) [reply source/tyr/setup.py:39]: See https://github.com/hove-io/navitia/pull/4017

**Pull request merged**